x/gasless/types/utils.go

```diff
@@ -0,0 +1,123 @@
+package types
+
+import (
+	"slices"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+// Comparable is a type constraint that allows only comparable types.
+type Comparable interface {
+	comparable
+}
```

Comment:
sorry it might be my go knowledge but why is this necessary?

Reply:
fixed